x-pack/plugins/siem/public/components/page/hosts/first_last_seen_host/index.test.tsx

```diff
@@ -162,7 +161,8 @@ describe('FirstLastSeen Component', async () => {
         </MockedProvider>
       </TestProviders>
     );
-    await wait(10);
-    expect(container.textContent).toBe(getEmptyValue());
+    await wait();
+    const alertIcon = await waitForElement(() => container.querySelectorAll('svg'), { container });
+    expect(alertIcon.length).toBe(1);
```

Comment:
Nice! I am hopeful this keeps the test deterministic and fast.

Reply:
Let's hope! I'll keep an eye on it over these next few days and remove it if it doesn't.